packages/sit-onyx/src/components/OnyxForm/OnyxForm.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { useDensity } from "../../composables/density";
+import { provideFormContext } from "./OnyxForm.core";
+import type { OnyxFormProps } from "./types";
+
+const props = defineProps<OnyxFormProps>();
+
+defineSlots<{
+  /**
+   * Any form content
+   */
+  default?(): unknown;
+}>();
+
+provideFormContext(props);
+
+const { densityClass } = useDensity(props);
+</script>
+<template>
```

Comment:
```suggestion
</script>

<template>
```

New line for my inner monk :D